### ISSUE TITLE
Ubuntu 20.04 expanded menu color wrong

### ISSUE THREAD

**Issue by apiontek** [rtlewis88/Solarized-Project#18]
I'm on Ubuntu 20.04, and when a dropdown menus' submenu expands, all items are colored active:

![Screenshot from 2020-04-28 12-00-00](https://user-images.githubusercontent.com/7383293/80509992-1c14d780-8948-11ea-9bbc-237682020013.png)

This doesn't seem right.

**Comment by rtlewis88** [comment]
You don't state what theme you are using; though it looks like you are experiencing a normal, albeit, annoying feature of any of the aggregate of current material themes. If so, the base architecture will not be altered - the material folks have their ways :(

Thanks for the feedback.

**Comment by apiontek** [comment]
Sorry, I didn't notice initially that this was a repo for multiple themes. It's Arc-Darkest, when set as both gnome-shell theme and applications/gtk theme.

**Comment by rtlewis88** [comment]
Arc-Darkest is not supposed to do that - and it has been tested tens of thousands of time, on our machines - and has never shown that. If you are using the wrong version, however, who knows what may be bugged out. I use Arc-Darkest Slate theme, which doesn't have the bleeding-edge updates yet. However, we have added the newest GNOME-3.36 versions, on several Arc-Darkest variants. If you are using the plain/classic Arc-Darkest variant, you can get versions for both **_pre_**-GNOME-3.36 or the very **_newest_** GNOME-3.36 - on four repos now - via Arc-Darkest, Arc-Darkest-COLORS, Arc-Black(the stand-alone w/classic blue-hue), or Arc-Black-COLORS. Arc is not messed up in any way, so just double check which one matches your operating system and you should be set.

**Comment by apiontek** [comment]
I don't know what to tell you -- I'm on Ubuntu 20.04 with gnome-shell 3.36. I had originally downloaded `Arc-Darkest-3.36_1.8.zip` from [gnome-look.org](https://www.gnome-look.org/p/1317409/), extracted it to /usr/share/themes/Arc-Darkest -- and that's it. Tried switching to it and got the result above.

Just to try, I just cloned the repo, did
```
git checkout -b Arc-Darkest origin/Arc-Darkest
sudo rm -rf /usr/share/themes/Arc-Darkest
sudo cp -R Arc-Darkest-3.36 /usr/share/themes/Arc-Darkest
```

And tried again ... same result.

I even tried in a fresh Ubuntu 20.04 VM, just in case I had some conflicting packages on my desktop. Same result.

I even tried in a fresh Ubuntu 18.04.3 VM, using the Arc-Darkest folder instead of the Arc-Darkest-3.36 folder -- and in fact, got the same coloring in expanding menus.

So it's not just that it's coloring like that in 20.04, it's doing it on 18.04 as well.

At this point I've moved on to Flat-Remix, so it's not that important to me, but I'd wanted to let you know.
